### PR TITLE
Issue 777 fix UI in mozilla firefox browser

### LINKED ISSUE #740: Fix: UI bug on Firefox
The token name and ID are overlapped on Firefox (Version 105.0.1 64-bit).
image

### LINKED ISSUE #777: Fix: UI in Mozilla firefox browser
# ℹ Overview
Fix issue with UI in Mozilla

### PR DESCRIPTION
**Pull request by IYTEC**
# ℹ Overview
Fix mozilla UI

### 📝 Related Issues
- closes #777 
- closes #740 

### 🔐 Acceptance:

- [ ] `yarn test` passes

### CODE REVIEW

File: components/Nav/Header/index.jsx [hunk 1]
```diff
@@ -1,50 +1,30 @@
-/*
```
Comment (atrefonas) [inline components/Nav/Header/index.jsx]: Please add the copyright back

Comment (IYTEC) [reply components/Nav/Header/index.jsx]: Done @atrefonas

**Review by atrefonas — CHANGES_REQUESTED** [review]
My open orders are not showing up, even when I refresh the page. I only see them when I go to a different tab (such as order history) and back. This was on Firefox.

WYWRYK42XADLY3O62N52BOLT27DMPRA3WNBT2OBRT65N6OEZQWD4OSH6PI is my wallet for the record.

image

**Review by atrefonas — CHANGES_REQUESTED** [review]
See comments

**Review by atrefonas — CHANGES_REQUESTED** [review]
Connect a wallet now looks incorrect on Chrome on my 14in Macbook. I don't see this problem on Testnet. I think it happened since you increased the font sizes.

image

**Comment by atrefonas** [comment]
Can you make reduce the font sizes in the top similar to how they were before? Doesn't have to be exactly the same. And make it so the Connect a Wallet doesn't appear on 3 lines and pops out of the box.

**Comment by atrefonas** [comment]
When wallet is connected looks like this:

image

**Comment by atrefonas** [comment]
Also I have 6 open orders but it isn't letting me scroll to see them.

image

**Review by atrefonas — APPROVED** [review]
Works pretty well

**Review by atrefonas — CHANGES_REQUESTED** [review]
When my pera wallet is connected in firefox, and I refresh, it crashes the browser window

**Comment by atrefonas** [comment]
> When my pera wallet is connected in firefox, and I refresh, it crashes the browser window

Also happens in Google Chrome

**Review by atrefonas — APPROVED** [review]

**Pull request merged**